### PR TITLE
Changed conditional to check for the word 'From'

### PR DESCRIPTION
**Pull request by AOrcilla**
using == instead of  .startswith()

### CODE REVIEW

File: dict-keys.py [hunk 1]
```diff
@@ -9,8 +9,8 @@
     lin = lin.strip()
     words = lin.split()
     for wrd in words: #Loop that reads through each word
-        if wrd.startswith("From:"): #Reads through split words to identify if the line starts with 'From:'
-            wrd = lin.strip()
+        if wrd == "From": #Reads through split words to identify if the line starts with 'From:'
+            wrd = lin.strip() #strips whitespace
```
Comment (dtil-gh) [inline dict-keys.py]: Delete this line, I believe it's unnecessary to strip spaces within a list that has already been stripped from line 9.

Comment (AOrcilla) [reply dict-keys.py]: Makes sense, deleting line 13.

**Review by AOrcilla — COMMENTED** [review]
Needs approval. @dtil-gh

**Comment by dtil-gh** [comment]
looks sexy

**Pull request merged**